docs/roadmap.md

```diff
@@ -0,0 +1,49 @@
+
+## I. :turtle: **Crawl** - Kick the tires and get a feel for the technology
+
+Your first few steps should be to provision a basic setup for FL and try to run a simple example. This will help you get a feel for the technology and the Azure ML platform. Great news, our [quickstart](./quickstart.md) will help you do just that!
+
+>**:checkered_flag: :turtle: Checkpoint:** You are able to reproduce a simple FL training run.
+
+## II. :walking: **Walk** - Provision a custom setup, connect non-sensitive data, train first FL model
+
+Now that you are familiar with the basics of FL on Azure ML, we recommend you train a FL model to solve your business problem, working with non-sensitive data at first. This will help you understand the different components of the FL training process, and how to customize them. And should you make a mistake with the security settings or inadvertently expose some data, that is okay because it is non-sensitive data.
+
+First, you will want to provision a FL setup relevant to your use case (internal geographically-distributed silos, external silos, etc...). Start with our [provisioning README](./provisioning/README.md) to identify which ingredients you will need, and provision your custom setup using our Bicep templates. Note that since you will not be working with sensitive data, you don't have to use the templates using VNets` and Private Endpoints. If you do though, it will make the **Run** phase a bit easier, since your setup will already have these safety measures applied - up to you.
+
+Next, you will want to connect your non-sensitive data to your FL setup, by creating a [data asset](https://learn.microsoft.com/en-us/azure/machine-learning/concept-azure-machine-learning-v2?tabs=cli#data). There are various ways to do so. For instance, you can manually upload data to your Azure ML workspace via the UI, or [use the CLI](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-data-assets?tabs=cli).
+
+After that, you are ready to start developing the training procedure for your business problem. Our repository proposes some [industry-relevant examples](./README.md#real-world-examples). If your use case is not too different from one of those examples, that would be a good starting point! If you would like to see a new industry-relevant example similar to your use case added to our repository, please [open a Feature Request](https://github.com/Azure-Samples/azure-ml-federated-learning/issues/new?assignees=&labels=&template=feature_request.md&title=) and we will consider enriching our corpus of examples.
+
+Once you have trained your FL model, we recommend comparing it to a baseline model trained on the central data. This will help you understand the impact of FL on your business problem, and assess whether you want to move forward.
+
+>**:checkered_flag: :walking: Checkpoint:** You have trained your first FL model and have demonstrated the feasibility of the approach.
```

Comment:
first FL model => + "relevant to your business goals"

the feasibility of the approach => + "and the value for your organization"

Reply:
done